### PR TITLE
Textarea validators

### PR DESCRIPTION
**Pull request by mjenny**
This fixes the issue with a blank screen when no validators.length is sent from BPME.

Additionally the error message is now properly displayed and I disabled the cacheDirectory for babel-loader.

### CODE REVIEW

**Comment by mjenny** [comment]
I still have to check why the prefilled value is not populated to the textarea.

**Comment by mjenny** [comment]
We have similar issues with the initial value not being displayed.

This might be due to the fact that we are defining textarea as a standard HTML element.

File: src/components/text-area/Textarea.js [hunk 1]
```diff
@@ -50,7 +54,7 @@ export default class Textarea extends PureComponent {
           className={classNames('a-text-area', {
             'has-error': meta.touched && meta.error,
           })}
-        />
+        >{value}</textarea>
```
Comment (AndyOGo) [inline src/components/text-area/Textarea.js]: React make stuff different for certain form elements:
https://reactjs.org/docs/forms.html#the-textarea-tag

File: src/renderers/ComponentRenderer.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ export default class ComponentRenderer extends PureComponent {
 
     const formValidators = []
     if (validators.required) {
-      formValidators.push(required())
+      formValidators.push(required(validators.required))
```
Comment (AndyOGo) [inline src/renderers/ComponentRenderer.js]: 👍

**Pull request merged**